src/site/stages/build/process-cms-exports/transformers/node-health_care_region_detail_page.js

```diff
@@ -0,0 +1,51 @@
+const {
+  getDrupalValue,
+  utcToEpochTime,
+  createMetaTagArray,
+} = require('./helpers');
+
+const transform = entity => ({
+  entityType: 'node',
+  entityBundle: 'health_care_region_detail_page',
+  title: getDrupalValue(entity.title),
+  changed: utcToEpochTime(getDrupalValue(entity.changed)),
+  entityPublished: getDrupalValue(entity.moderationState) === 'published',
```

Comment:
I just made [this](https://github.com/department-of-veterans-affairs/vets-website/pull/11497/files?file-filters[]=.js#diff-1d5006db7b1dc3569634f6f4a03e526fR151-R153) in my last PR, for what it's worth. To limit our magic string usage.